src/popup/pages/TransactionDetails.vue

```diff
@@ -21,7 +21,10 @@
         />
       </div>
       <div class="content">
-        <TransactionOverview v-bind="transaction" />
+        <TransactionOverview
+          v-bind="transaction"
```

Comment:
Maybe you could change this to regular prop? Binding an object into component props this way is a really terrible thing.
If you don't want to extend the PR skip and resolve my comment.